main/kcal-culator.py

```diff
@@ -0,0 +1,44 @@
+#[x] create a program that take a file location
+#[x] opens and reads the file (displays all the content on the terminal)
+#[x] if no file, promps a message no file
+#[x] asks if to create or quit
+#[x] creates or quits
+
+def gather_location(): 
+    user_input = input("To open a data sheet specify file location.\nEnter \"default\" for the default file.\n") 
+    user_input.strip()
+    #add some string processing to clear the input
+
+    if user_input.lower() == "default": 
+        user_input = "/Users/martapodziewska/Documents/01_Coding/01_GitHub_working_repos/kcal-culator/main/draft_data_sheet.csv"
```

Comment:
Consider setting the default file to be relative to the project folder ... usually at the root of the folder. 
This will currently fail on any non `martapodziewska` computer :)

Reply:
thanks, good point! I tried to make this work at the beginning, but when I would run the program with my terminal in another folder (giving a path to run it) it was not working. Do you know why?

Reply:
Hmm ... it might depend on how you're providing the path to the file. Can you try specifying it as `./the-file.csv`?